packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -1,45 +1,138 @@
 <template>
-    <div
-        :class="$style['o-link']"
-        data-test-id="link"
-    >
-        {{ copy.text }}
-    </div>
+    <a
+        :class="[
+            $style['o-link'],
+            { [$style['o-link--bold']]: isBold },
+            { [$style['o-link--noDecoration']]: !hasTextDecoration },
+            { [$style['o-link--full']]: isFullWidth }
+        ]"
+        :data-test-id="`${linkText} link`"
+        :href="url"
+        :target="target"
+        :aria-label="ariaLabel"
+        :rel="rel"
+    >{{ linkText }}</a>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
+import { VueGlobalisationMixin } from '@justeat/f-globalisation';
 import tenantConfigs from '../tenants';
 
 export default {
     name: 'VLink',
-    components: {},
+
+    mixins: [VueGlobalisationMixin],
+
     props: {
         locale: {
             type: String,
-            default: ''
+            default: 'en-GB'
+        },
+
+        linkText: {
+            type: String,
+            required: true
+        },
+
+        url: {
+            type: String,
+            required: true
+        },
+
+        isExternal: {
+            type: Boolean,
+            default: false
+        },
+
+        opensInNewLocation: {
+            type: Boolean,
+            default: false
+        },
+
+        isBold: {
+            type: Boolean,
+            default: false
+        },
+
+        hasTextDecoration: {
+            type: Boolean,
+            default: true
+        },
+
+        isFullWidth: {
+            type: Boolean,
+            default: false
         }
     },
-    data () {
-        const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
-        const localeConfig = tenantConfigs[locale];
 
+    data () {
         return {
-            copy: { ...localeConfig }
+            tenantConfigs
         };
+    },
+
+    computed: {
+        ariaLabel () {
+            let message;
+
+            if (this.isExternal) {
+                message = this.$t('ariaLabel.externalSite');
+            } else if (this.opensInNewLocation) {
+                message = this.$t('ariaLabel.newLocation');
+            }
+
+            return message ? this.linkText + message : this.linkText;
```

Comment:
I think we need to add ariaLabel only if we need to add info about link opening in a new location and not duplicate just link text in case if it opens in the same page